### PR TITLE
Make sure values in Json are well-formed

### PR DESCRIPTION
**Pull request by regiskuckaertz**
Take a look at [this](http://api.nextgen.guardianapps.co.uk/au/lifeandstyle/relationships/lite.json) and [the `EditorsPick` type](https://github.com/guardian/content-api/blob/a25c8fdbcda47451ea56a7a1f6ba7fbeda4a883f/porter/src/main/scala/com.gu.contentapi.porter/model/EditorsPicks.scala).

Some items in the JSON are missing information, if you scroll down you will see some missing the `trailText`. As a result, we are unable to index editors' picks for all fronts exposing this behaviour (our logs show loads).

It looks like the pressed content is outdated, because in the example above if you query CAPI you will see they are complete. There might be an issue upstream, but here I'm just making sure the content in the JSON is well-formed so that deserialisation will not fail on our end.

### CODE REVIEW

File: facia/app/controllers/front/FapiFrontJsonLite.scala [hunk 1]
```diff
@@ -21,10 +21,12 @@ trait FapiFrontJsonLite {
         "displayName" -> pressedCollection.displayName,
         "href" -> pressedCollection.href,
         "id" -> pressedCollection.id,
-        "content" -> pressedCollection.curatedPlusBackfillDeduplicated.filterNot(isLinkSnap).map(getContent))
+        "content" -> pressedCollection.curatedPlusBackfillDeduplicated.filterNot(isLinkSnap).map(getContent).filter(isComplete))
       .fields
       .filterNot{ case (_, v) => v == JsNull})
 
+  private def isComplete(json: JsValue): Boolean =
```
Comment (TBonnin) [inline facia/app/controllers/front/FapiFrontJsonLite.scala]: can we rename it something a bit more meaninful like `hasAllMandatoryKeys`?

**Review by TBonnin — COMMENTED** [review]
It would be better to figure out what is the source of those imcomplete json and fix the underlying issue. No?

**Comment by regiskuckaertz** [comment]
I'm not familiar with `PressedContent`, can we pair and see if we can find the source of the issue?

**Comment by TBonnin** [comment]
@regiskuckaertz yes of course

**Comment by regiskuckaertz** [comment]
@TBonnin can we give this one another try?

**Comment by regiskuckaertz** [comment]
long story short: I could use your help for testing 😂

**Comment by regiskuckaertz** [comment]
@TBonnin are you happy for me to merge?

**Review by TBonnin — APPROVED** [review]

**Pull request merged**